client/my-sites/activity/activity-log-v2/index.tsx

```diff
@@ -39,12 +42,28 @@ const ActivityLogV2: FunctionComponent = () => {
 
 	const jetpackCloudHeader = siteHasFullActivityLog ? (
 		<div className="activity-log-v2__header">
-			<h2>{ translate( 'Find a backup or restore point' ) }</h2>
-			<p>
-				{ translate(
-					'This is the complete event history for your site. Filter by date range and/or activity type.'
+			<div className="activity-log-v2__header-left">
+				<div className="activity-log-v2__header-title">{ translate( 'Activity log' ) }</div>
+				<div className="activity-log-v2__header-text">
+					{ translate( 'This is the complete event history for your site' ) }
+				</div>
+			</div>
+			<div className="activity-log-v2__header-right">
+				{ selectedSiteSlug && (
+					<Tooltip
+						text={ translate(
+							'To test your site changes, migrate or keep your data safe in another site'
+						) }
+					>
+						<Button
+							className="activity-log-v2__clone-button"
+							href={ backupClonePath( selectedSiteSlug ) }
+						>
+							{ translate( 'Copy this site' ) }
+						</Button>
+					</Tooltip>
```

Comment:
Not a blocker, but maybe we could consider making this reusable, as it is also required on the backup page and just differs on `className`.